work/prediction-notes.ipynb

```diff
@@ -0,0 +1,134 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# COVIDvu <img src='resources/UN-flag.png' align='right'>\n",
+    "\n",
+    "COVID-19 model for predicting future outcomes.\n",
```

Comment:
Would this apply here?

_COVID-19 model for predicting the rate at which the virus burn through the population._

Or something like that?

Reply:
"Burn" as in epidemiology jargon for a virus epidemic infecting all suceptinble subjects until it runs out.

Reply:
How about

_COVID-19 model for predicting the spread of coronavirus through populations._?

Struggled to make a title with the word "burn" that wasn't too verbose.